### PR TITLE
Update Video Downloader 

### PR DESCRIPTION
**Pull request by GustavoPeredo**
Hello! I really like the app Video Downloader, but recently the current maintainer, Unrud, archived the main repository and removed youtube-dl as a dependency (meaning the app is currently broken on flathub). I want to keep maintaining the app. I also added a dark mode :)

### CODE REVIEW

**Comment by GustavoPeredo** [comment]
bot, build com.github.unrud.VideoDownloader

**Comment by GustavoPeredo** [comment]
bot, build com.github.unrud.VideoDownloader

File: com.github.unrud.VideoDownloader.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
+    "app-id" : "com.github.unrud.VideoDownloader",
```
Comment (nedrichards) [inline com.github.unrud.VideoDownloader.json:2]: if this is a new app that you plan to maintain (albeit forked from videodownloader) then you need to change the app-id to match.

Comment (GustavoPeredo) [reply com.github.unrud.VideoDownloader.json:2]: My original idea was to keep updating video-downloader that is already on Flathub, would that be possible?

Comment (bilelmoussaoui) [reply com.github.unrud.VideoDownloader.json:2]: Except that's not the same app at all but a fork of it. So please use a different app-id in such case.

Comment (GustavoPeredo) [reply com.github.unrud.VideoDownloader.json:2]: Understandable, will do so

**Comment by GustavoPeredo** [comment]
bot, build org.gustavoperedo.VideoDownloader

**Comment by GustavoPeredo** [comment]
I changed the app-id

File: org.gustavoperedo.VideoDownloader.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "command" : "video-downloader",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.FileManager1",
+        "--filesystem=xdg-download/VideoDownloader:create",
+        "--metadata=X-DConf=migrate-path=/org/gustavoperedo/VideoDownloader/"
```
Comment (barthalion) [inline org.gustavoperedo.VideoDownloader.json:14]: I don't think it's needed. It's not as if you were using this name previously so there's nothing to migrate.

File: org.gustavoperedo.VideoDownloader.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
+{
+    "app-id" : "org.gustavoperedo.VideoDownloader",
```
Comment (barthalion) [inline org.gustavoperedo.VideoDownloader.json:2]: Do you own this domain? If not, it should be `com.github.GustavoPeredo.VideoDownloader`.

Comment (GustavoPeredo) [reply org.gustavoperedo.VideoDownloader.json]: I changed it to io.github.gustavoperedo.VideoDownloader (following the [documentation](https://docs.flatpak.org/en/latest/conventions.html#application-ids).)

I have another app that I used the org.gustavoperedo prefix, for me to update it, do I just have to change the manifest's app-id? (The app is already on flathub)

**Comment by GustavoPeredo** [comment]
bot, build io.github.gustavoperedo.VideoDownloader

File: org.gustavoperedo.VideoDownloader.json (hunk 3 above)
Comment (barthalion) [reply org.gustavoperedo.VideoDownloader.json]: No, it's fine for existing app to stay that way. We try to limit friction caused by ID changes to minimum.

**Pull request closed without merging**